telegram/request/_requestparameter.py

```diff
@@ -143,7 +143,7 @@ def from_input(cls, key: str, value: object) -> "RequestParameter":
         """Builds an instance of this class for a given key-value pair that represents the raw
         input as passed along from a method of :class:`telegram.Bot`.
         """
-        if isinstance(value, list):
+        if not isinstance(value, (str, bytes)) and isinstance(value, Sequence):
```

Comment:
We'll need new unit tests that ensure that

1. bytes and str are in fact not treated as sequence
2. different types of sequences work, e.g. `list`, `tuple` and also custom subclasses of `collections.abc.Sequence`

Can you add corresponding tests to `test_requestparameter.py`? Maybe `test_from_input_str_and_bytes` and `test_from_input_different_sequences`?